### ISSUE TITLE
Inquiry regarding the Settings of Hyper-parameters

### ISSUE THREAD

**Issue by sap01** [flatironinstitute/inferelator_sc#33]
Hello Bonneau Lab members,

I am a member of the Roy Lab at UW-Madison. At our lab, we are trying to utilize the multi-task workflow of AMuSR. In that regard, there is a small doubt. How can we define the values of the hyper-parameters (such as the regularization penalties) in the input file? 

Regards,
Saptarshi

**Comment by asistradition** [comment]
Sorry for the delay. 

This option was buried in some code refactoring I hadn't gotten around to finishing. It's now available in v0.5.2 (which is on pypi).

You can [set the regression penalties](https://inferelator.readthedocs.io/en/latest/model_selection.html#inferelator.regression.amusr_regression.AMUSRRegressionWorkflowMixin.set_regression_parameters) at runtime as follows:

```
from inferelator import inferelator_workflow
import numpy as np

workflow = inferelator_workflow(workflow="amusr", regression="amusr")
workflow.set_regression_parameters(lambda_Bs=np.arange(10), lambda_Ss=np.arange(10))
```

Please let me know if there are any problems.

**Comment by sap01** [comment]
Thank you very much for the clarification!

Seems like there shouldn't be any problems. If any, I'll reach out to you.
